src/pages/iou/request/step/IOURequestStepConfirmation.js

```diff
@@ -86,6 +81,7 @@ function IOURequestStepConfirmation({
     const {translate} = useLocalize();
     const {windowWidth} = useWindowDimensions();
     const {isOffline} = useNetwork();
+    const personalDetails = usePersonalDetails() || CONST.EMPTY_OBJECT;
```

Comment:
All changes in this file come from https://github.com/Expensify/App/pull/31076  (follow guideline: Replace all uses of the Old Component with the New Component)